domains/README.md

```diff
@@ -1,16 +1,73 @@
-# Cumulus fork
-
-This is a fork of [cumulus](https://github.com/paritytech/cumulus/commit/520012619e5428be50f47917aacd54e47860f43e), an experiment of the groundwork of Subspace secondary chains framework.
-
-- `client`
-  - `cirrus-executor`
-  - `cirrus-service`
-  - `cli`
-  - `consensus`
-    - `common`
-    - `relay-chain`
-- `parachain-template`
-  - `node`
-  - `runtime`
-
-The binary `parachain-collator` was renamed to `subspace-executor`.
+# Domain operator
+
+Reference implementation of Domain Operator for Subspace Network Blockchain.
+
+## Overview
+
+Domain is the decouple execution layer of the [Subspace Network Blockchain](https://subspace.network) (referred as consensus chain below).
+
+The extrinsic of the domain chain is first collected into a bundle by the domain operator, the bundle is then broadcast to the consensus chain network as an extrinsic of the consensus chain.
+
+The domain operator listening to the block import event of the consensus chain, extract bundles from the imported consensus block and execute the extrinsic of the bundle to build and import a domain block in a deterministically way.
+
+NOTE: currently, the domain chain is not support to sync from other operator node and need to deterministically deriving from the consensus chain block by block.
+
+### Build from source
+
+The domain operator node is embeded within the `subspace-node` binary, please refer to [Subspace node](../crates/subspace-node/README.md) for how to build from source.
+
+### Start the domain operator node
+
+The domain operator node is running with an embeded consensus node, thus you need to specified the args for both the consensus node and the domain operator node:
+
+```bash
+subspace-node [consensus-chain-args] -- [domain-args]
+```
+
+Example:
+
+Start a single node development chain:
+```bash
+target/production/subspace-node \
+    --dev \
+    --rpc-external \
+    --node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+    -- \
+    --domain-id 0 \
```

Comment:
Right, it needs to be the same, domain 0 (the EVM domain) is the only domain instance we support from genesis. Though advanced users can instantiate and run their own domain instance, I'm hesitant if we should add a section about this here, since it will bring some technical detail.

BTW in both examples, the `domain-id` is passing the the domain operator instead of the consensus chain.